classes/patreon_routing.php

```diff
@@ -189,7 +247,10 @@ function parse_request( &$wp ) {
 						
 					}
 		
-					if( $patreon_level == 0 ) {
+					if( $patreon_level == 0  
+						AND !( isset($GLOBALS['patreon_enable_direct_unlocks']) 
+								AND $GLOBALS['patreon_enable_direct_unlocks'] )
```

Comment:
The formatting of these AND blocks seems a bit weird

Reply:
Yeah, they do seem weird - the lower two lines are actually a grouped condition. When indented, its weird. I made them into one line to see how they will play out.